### PR TITLE
new resource: `azurerm_kubernetes_fleet_update_run`

### PR DESCRIPTION
**Pull request by ms-henglu**

### CODE REVIEW

File: website/docs/r/kubernetes_fleet_update_run.html.markdown [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    upgrade {
+      type               = "Full"
+      kubernetes_version = "1.27"
+    }
+    node_image_selection {
+      type = "Latest"
+    }
+  }
+  fleet_update_strategy_id = azurerm_kubernetes_fleet_update_strategy.example.id
+}
```
Comment (magodo) [inline website/docs/r/kubernetes_fleet_update_run.html.markdown:55]: Should this also have the `stage` be included in the `ignore_changes`?

File: website/docs/r/kubernetes_fleet_update_run.html.markdown [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Kubernetes Fleet Update Run. Changing this forces a new Kubernetes Fleet Update Run to be created.
+
+* `kubernetes_fleet_manager_id` - (Required) The ID of the Fleet Manager. Changing this forces a new Kubernetes Fleet Update Run to be created.
+
+* `managed_cluster_update` - (Required) A `managed_cluster_update` block as defined below.
+
+* `fleet_update_strategy_id` - (Optional) The ID of the Fleet Update Strategy. Only one of `fleet_update_strategy_id` or `fleet_update_strategy` can be specified.
```
Comment (magodo) [inline website/docs/r/kubernetes_fleet_update_run.html.markdown:68]:
```suggestion
* `fleet_update_strategy_id` - (Optional) The ID of the Fleet Update Strategy. Only one of `fleet_update_strategy_id` or `stage` can be specified. 
```

**Review by magodo — CHANGES_REQUESTED** [review]
Thanks for this PR!
I have left some additional mostly minor comments that once addressed this should be good to merge 👍

File: website/docs/r/kubernetes_fleet_update_run.html.markdown (hunk 1 above)
Comment (ms-henglu) [reply website/docs/r/kubernetes_fleet_update_run.html.markdown:74]: Thanks for the suggestion, I've updated it to use a more recommended example.

**Comment by magodo** [comment]
LGTM!

**Review by katbyte — APPROVED** [review]
LGTM ⛵

**Pull request merged**